app/commands.py

```diff
@@ -534,8 +534,8 @@ async def _with(ctx: Context) -> Optional[str]:
 @command(Privileges.UNRESTRICTED, aliases=["req"])
 async def request(ctx: Context) -> Optional[str]:
     """Request a beatmap for nomination."""
-    if ctx.args:
-        return "Invalid syntax: !request"
+    if len(ctx.args) < 1:
+        return "Invalid syntax: !request <ranked/loved> [optional comment]"
```

Comment:
re. `<ranked/loved>`, i think there's limited value in having users able to request a status, i think having the bn team decide will be fine for most servers.

on akatsuki i've always thought it's best to keep the input unbiased and just provide the bn with maps to look at - where they decide it should be categorized (ranked, loved, etc.) is ultimately up to the team imo, so i'm also not sure of the value of `comment`.

i think something more valuable could be having the the team be able to provide feedback for why a request was declined, such that in those cases the downstream users who made the requests have some understanding of how to better nominate maps in the future